grammar.js

```diff
@@ -61,11 +62,15 @@ module.exports = grammar({
   word: $ => $.identifier,
 
   rules: {
-    compilation_unit: $ => repeat($._definition),
+    compilation_unit: $ => repeat($._top_level_definition),
 
-    _definition: $ => choice(
+    _top_level_definition: $ => choice(
       $.package_clause,
       $.package_object,
+      $._definition,
+    ),
+
+    _definition: $ => choice(
```

Comment:
this is the second and final _meaningful change_ in this PR, the rest is just to make it compile